### ISSUE TITLE
Config default lang update

### ISSUE THREAD

**Issue by eknowlton** [lopes/zola.386#4]
I love your theme. I'm going to use it. Thank you for putting it together.

I went to clone the repo, and serve the site with `zola serve` but ran into a issue.

```
 ✘ ethan@edison  ~/Projects
 ➜ git clone https://github.com/lopes/zola.386 knowlton-dev
Cloning into 'knowlton-dev'...
remote: Enumerating objects: 166, done.
remote: Counting objects: 100% (166/166), done.
remote: Compressing objects: 100% (125/125), done.
remote: Total 166 (delta 60), reused 130 (delta 35), pack-reused 0
Receiving objects: 100% (166/166), 1.05 MiB | 1.64 MiB/s, done.
Resolving deltas: 100% (60/60), done.
 ethan@edison  ~/Projects
 ➜ cd knowlton-dev
 ethan@edison  ~/Projects/knowlton-dev   master
 ➜ zola serve
Building site...
Checking all internal links with anchors.
> Successfully checked 0 internal link(s) with anchors.
-> Creating 3 pages (1 orphan), 0 sections, and processing 1 images
1 page(s) ignored (missing date or weight in a sorted section):
- /home/ethan/Projects/knowlton-dev/content/unlisted.md
Error: Failed to render page '/home/ethan/Projects/knowlton-dev/content/sem-nulla.md'
Reason: Failed to render 'page.html' (error happened in 'index.html').
Reason: Variable `config.default_language` not found in context while rendering 'page.html'
```

I found this issue with https://github.com/getzola/zola/issues/1442 

Seems `config.default_language` is asscess by just `lang` now?

I replaced `config.default_language` with `lang` and it built and served as expected.

`<html lang="{% if page.extra.lang %}{{ page.extra.lang }}{% else %}{{ config.default_lang }}{% endif %}" itemscope itemtype="http://schema.org/Blog">`langlang

`<html lang="{% if page.extra.lang %}{{ page.extra.lang }}{% else %}{{ lang }}{% endif %}" itemscope itemtype="http://schema.org/Blog">`

**Comment by eknowlton** [comment]
Fixed by #5

**Comment by lopes** [comment]
Fixed with new PR.
